utils/ssh.py

```diff
@@ -325,14 +356,42 @@ def put_file(self, local_file, remote_file='.', **kwargs):
             return scp
 
     def get_file(self, remote_file, local_path='', **kwargs):
-        logger.info("Transferring remote file {remote_file} to local {local_path}".format(
-            remote_file=remote_file, local_path=local_path))
+        logger.info("Transferring remote file %r to local %r", remote_file, local_path)
+        base_name = os_path.basename(remote_file)
         if self.is_container:
-            tempfilename = '/share/temp_{}'.format(fauxfactory.gen_alpha())
+            tmp_file_name = 'temp_{}'.format(fauxfactory.gen_alpha())
+            tempfilename = '/share/{}'.format(tmp_file_name)
             self.run_command('cp {} {}'.format(remote_file, tempfilename))
             scp = SCPClient(self.get_transport(), progress=self._progress_callback).get(
                 tempfilename, local_path, **kwargs)
             self.run_command('rm {}'.format(tempfilename))
+            call([
+                'mv',
+                os_path.join(local_path, tmp_file_name),
+                os_path.join(local_path, base_name)])
+            return scp
+        elif self.is_pod:
+            tmp_folder_name = 'automation-{}'.format(fauxfactory.gen_alpha().lower())
+            tmp_file_name = 'file-{}'.format(fauxfactory.gen_alpha().lower())
+            tmp_full_name = '/tmp/{}/{}'.format(tmp_folder_name, tmp_file_name)
+            # Clean up container's temporary folder
```

Comment:
**Optional**
Consider logging these folder and file names for later debugging, to associate the random names with unique test runs.